src/DeployPage.js

```diff
@@ -94,7 +98,7 @@ class DeployPage extends React.Component {
         });
       }
     } catch (error) {
-      console.warn('Error fetching shas!');
+      console.warn('Error fetching deployment /shas or tags!');
```

Comment:
@janbrasna is that `/` needed? If so, might benefit from a space after it?

```suggestion
      console.warn('Error fetching deployment shas or tags!');
```

Reply:
Oh, the `/shas` is an actual endpoint;) It's just a console warning, so can't really be formatted to make that clear; but at the same time I wanted to mention it's an endpoint failing — that may or may not have anything to do with the actual deployments;)

Would rearranging help with language?

```suggestion
      console.warn('Error fetching /shas deployment or tags!');
```

_(maybe the pun reusing the endpoint name within the sentence wasn't as smart as I thought;D)_

Reply:
D'oh, I see what you mean :)